### PR TITLE
Add example Plotter.remove_legend()

### PR DESCRIPTION
**Pull request by ENIAC-6**
Add example Plotter.remove_legend()

### Overview

Add example Plotter.remove_legend()

Issue #1629

### Details

Improve example coverage of plotting module.

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -849,9 +849,28 @@ def add_legend(self, *args, **kwargs):
...
+        >>> from pyvista import examples
+        >>> sphere = pyvista.Sphere(center=(0, 0, 1))
+        >>> cube = pyvista.Cube()
+        >>> pl = pyvista.Plotter()
+        >>> _ = pl.add_mesh(sphere, 'grey', smooth_shading=True, label='Sphere')
+        >>> _ = pl.add_mesh(cube, 'r', label='Cube')
+        >>> _ = pl.add_legend(bcolor='w', face=None)
+        >>> pl.show()
 
+        >>> pl.remove_legend()
```
Comment (akaszynski) [inline pyvista/plotting/plotting.py:869]: It's not possible to close and then reopen a plotter. You will probably just want to:

```suggestion
        >>> pl.remove_legend()
```

Comment (tkoyama010) [reply pyvista/plotting/plotting.py:869]: @akaszynski Thanks. @ENIAC-6  Could you click `Add suggestion to batch` and `commit changes` ?

**Comment by akaszynski** [comment]
Any method that uses `@wraps(...)` uses the docstring from the method it's wrapping. This means that the docstring of `remove_legend` is actually already written:
https://github.com/pyvista/pyvista/blob/2c52ffda4a0cf099c2c8950677492542698e4642/pyvista/plotting/renderer.py#L3002-L3009

Let's close this out, but keep up all the PRs!

**Comment by tkoyama010** [comment]
Oh I didn't know that. Sorry, @ENIAC-6 it's my fault.

**Pull request closed without merging**